packages/core/src/Cardano/types/Script.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-use-before-define */
+
+import { Slot } from './';
+
+export declare type ScriptPlutus = string;
+
+export interface PlutusV1 {
+  'plutus:v1': ScriptPlutus;
+}
+export interface PlutusV2 {
+  'plutus:v2': ScriptPlutus;
+}
+
+export interface ExpiresAt {
+  expiresAt: Slot;
+}
+export interface StartsAt {
+  startsAt: Slot;
+}
+
+export declare type DigestBlake2BVerificationKey = string;
+
+export declare type Script = Native | PlutusV1 | PlutusV2;
+export declare type ScriptNative = DigestBlake2BVerificationKey | Any | All | NOf | ExpiresAt | StartsAt;
```

Comment:
Let's make sure to use consistent terminology
```suggestion
export declare type ScriptSimple = DigestBlake2BVerificationKey | Any | All | NOf | ExpiresAt | StartsAt;
```

Reply:
I will update it once we merge the other branch where we are introducing the new type

Reply:
This is now a non-issue right?

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/946ca05a728386d06962959c2997dd07059329dd